ged2dot.py

```diff
@@ -840,7 +840,6 @@ def load(self) -> None:
                         elif self.inDeat:
                             self.indi.deat = year
 
-            # pylint: disable=broad-except
```

Comment:
Don't remove this, it was there on purpose to document that catching `Exception` is OK in this particular case, even if it's a bad idea in general.

Reply:
oops. That was purely unintentional. Not sure how I did that.